src/include/duckdb/main/extension_entries.hpp

```diff
@@ -810,8 +816,6 @@ static constexpr ExtensionFunctionOverloadEntry EXTENSION_FUNCTION_OVERLOADS[] =
     {"generate_series", "core_functions", CatalogType::SCALAR_FUNCTION_ENTRY, "BIGINT>BIGINT[]"},
     {"generate_series", "core_functions", CatalogType::SCALAR_FUNCTION_ENTRY, "BIGINT,BIGINT>BIGINT[]"},
     {"generate_series", "core_functions", CatalogType::SCALAR_FUNCTION_ENTRY, "BIGINT,BIGINT,BIGINT>BIGINT[]"},
-    {"generate_series", "core_functions", CatalogType::SCALAR_FUNCTION_ENTRY,
-     "TIMESTAMP,TIMESTAMP,INTERVAL>TIMESTAMP[]"},
```

Comment:
I am not sure I see why this is removed / what's special about this function.

Reply:
I don't know... cc @Mytherin I ran `python3 scripts/generate_extensions_function.py --extensions spatial
` what am I doing wrong?

Reply:
I don't know

Reply:
There's probably a bug in the script - it should work with an individual extension as well

Reply:
I think I also experienced this today, it somehow doesn't check for existing entries in `EXTENSION_FUNCTIONS`, so it gets put there instead of in `EXTENSION_FUNCTION_OVERLOADS`

Reply:
I am already messing with the script a bit for <https://github.com/duckdb/duckdb/pull/16304>, I'll see if I can include the fix for this problem in there

Reply:
I think I've found the issue
For functions like `age`, that exist in both `icu` as well as `core_functions`, there is a condition that triggers to remove the function from the `FUNCTIONS` map and add it to `OVERLOADS` map instead, but that won't trigger because the existing entries are added later

I *think* just adding the existing entries first should fix this